src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -43,6 +43,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <UseSystemResourceKeys Condition="$(IlcDisableReflection) == 'true'">true</UseSystemResourceKeys>
```

Comment:
Could you delete the PublishTrimmed on line 28 while you're on it? I noticed in your SDK PR that we set this to true on the SDK side so this should not be needed.